### PR TITLE
Dark Mode: Fix some colors

### PR DESCRIPTION
**Pull request by GuillaumeRx**
Fix some random colors remaining in the codebase

### CODE REVIEW

File: ui/pages/create-account/connect-hardware/index.scss [hunk 1]
```diff
@@ -186,7 +186,7 @@
   }
 
   &__item:hover {
-    background-color: rgba(0, 0, 0, 0.06);
+    background-color: var(--color-overlay-default);
```
Comment (georgewrmarshall) [inline ui/pages/create-account/connect-hardware/index.scss:189]: We need to add a pattern for tables so we don't really have this. Let's update the table to not have alternating backgrounds and instead add a bottom border to `&__item`
```
  &__item {
    @include Paragraph;

    flex-direction: row;
    display: flex;
    padding-left: 10px;
    padding-right: 10px;
    border-bottom: 1px solid var(--color-border-muted); // Adding border here
  }
```

Then on hover we can use background-alternative..

```suggestion
    background-color: var(--color-background-alternative);
```

**Review by darkwing — DISMISSED** [review]

File: ui/pages/swaps/exchange-rate-display/exchange-rate-display.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default function ExchangeRateDisplay({
   secondaryTokenValue,
   secondaryTokenDecimals = 18,
   secondaryTokenSymbol,
-  arrowColor = 'black',
+  arrowColor = 'var(--color-text-default)',
```
Comment (georgewrmarshall) [inline ui/pages/swaps/exchange-rate-display/exchange-rate-display.js:15]: Would we get away with `var(--color-icon-default)` here?

File: ui/pages/unlock-page/unlock-page.component.js [hunk 3]
```diff
@@ -129,7 +129,7 @@ export default class UnlockPage extends Component {
   renderSubmitButton() {
     const style = {
       backgroundColor: 'var(--color-primary-default)',
-      color: 'white',
+      color: 'var(--color-primary-inverse)',
```
Comment (georgewrmarshall) [inline ui/pages/unlock-page/unlock-page.component.js:132]: Nice catch!

**Review by georgewrmarshall — COMMENTED** [review]
Nice work! Made one suggestion regarding the hardware wallet table


https://user-images.githubusercontent.com/8112138/160431317-085f65bf-59fa-4e25-b030-ce6f4abf32f4.mov

File: ui/pages/create-account/connect-hardware/index.scss (hunk 1 above)
Comment (georgewrmarshall) [reply ui/pages/create-account/connect-hardware/index.scss:189]: Then we can remove these styles

```
  &__item:nth-of-type(even) {
    background-color: var(--color-background-default);
  }
  &__item:nth-of-type(odd) {
    background: var(--color-background-alternative);
  }
```

**Review by darkwing — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**